hrldas/HRLDAS_forcing/run/examples/ERA5/Prepare_ERA5-Land_Forcing.sh

```diff
@@ -19,11 +19,18 @@
 #                                                                                               #
 #################################################################################################
 
-year=1980
-INFILE=/fas4/anav/INPUTS/ERA5_for_Noah-MP/YEARLY/ERA5-Land_Noah-MP_$year.grb
+# set date for data processing
+inyear=2023
+inmonth=11
+inday=01
+preyear=2023
+premonth=10
+preday=31
 
-let last_year=$year-1
-PREVIOUS_YEAR=/fas4/anav/INPUTS/ERA5_for_Noah-MP/YEARLY/ERA5-Land_Noah-MP_$last_year.grb
+indate=${inyear}-${inmonth}-${inday}
+INFILE=../../../ERA5_forcing/ERA5-Land-Noah-MP-${indate}.grib
```

Comment:
this needs to be INFILE=../../../ERA5_forcing/ERA5-Land-Noah-MP-${inyear}_${inmonth}_${inday}.grib, or you can rename your ERA5 input data file name. Similarly for the PREVIOUS_FILE.